specs/http-client/index.html

```diff
@@ -429,117 +428,116 @@ <h3>Serializing the request content</h3>
           The <code>http:header</code> elements are as defined for the request, but represent
-          instead the response headers.  The <code>http:body</code>
-          and <code>http:multipart</code> elements are also like in the request, but
+          instead the response headers. The <a>http:body</a>
+          and <a>http:multipart</a> elements are also like in the request, but
           <code>http:body</code> elements must be empty.</p>
       </section>
 
       <section>
-        <h3>Representing the result content</h3>
-        <p>Instead of being inserted within the <code>http:response</code> element, the content of
-          each body is returned as a single item in the return sequence.  Each item is in the same
-          order (after the <code>http:response</code> element) than the <code>http:body</code>
-          elements.  For each body, the way this item is built from the HTTP response is as
+        <h3>The Response Entity Body</h3>
+        <p>Instead of being inserted within the <a>http:response</a> element, the content of
+          each body is returned as a single item in the returned sequence. Each item is in the same
+          order (after the <a>http:response</a> element) as the <a>http:body</a>
+          elements. For each body, the way this item is built from the HTTP response is as
           follow.</p>
         <p>If the <code>status-only</code> attribute has the value <code>true</code> (default
           is <code>false</code>), the returned sequence will only contain the
-          <code>http:response</code> element (with the headers, but also the empty
-          <code>http:body</code> or <code>http:multipart</code> elements, as if
-          <code>status-only</code> was false), and the following items, representing the bodies
+          <a>http:response</a> element (with the headers, but also the empty
+          <a>http:body</a> or <a>http:multipart</a> elements, as if
+          <code>status-only</code> was <code>false</code>), and the following items, representing the bodies
           content are not generated from the HTTP response.</p>
-        <p>For each body that has to be interpreted, the following rules apply in order to build the
-          corresponding item.  If the body media type is a text media type, the item is a string,
-          containing the body content.  If the media type is an XML media type, the content is
-          parsed and the item is the resulting document node.  If the media type is an HTML type,
+        <p>For each body that has to be parsed, the following rules apply in order to build the
+          corresponding XDM item. If the body media type is a text media type, the item is an <code>xs:string</code>,
+          containing the body content. If the media type is an XML media type, the content is
+          parsed and the item is the resulting XDM <code>document-node()</code>. If the media type is an HTML type,
```

Comment:
I would keep "document node" rather than the more complicated (and in my view unnecessary) "XDM `document-node()`".

Reply:
Okay I changed it back.